tests/sentry/lang/native/test_symbolicator.py

```diff
@@ -265,3 +271,135 @@ def test_reverse_aliases(self, builtin_sources):
         reverse_aliases = symbolicator.reverse_aliases_map(builtin_sources)
         expected = {"sentry:ios-source": "sentry:ios", "sentry:tvos-source": "sentry:ios"}
         assert reverse_aliases == expected
+
+
+class TestIgnoredSourcesFiltering:
+    @pytest.fixture
+    def sources(self):
+        builtins = [
+            {
+                "id": "sentry:microsoft",
+                "name": "Microsoft",
+                "type": "gcs",
+            },
+            {
+                "id": "sentry:electron",
+                "name": "Electron",
+                "type": "s3",
+            },
+            {
+                "id": "sentry:ios-source",
+                "name": "iOS",
+                "type": "http",
+            },
+            {
+                "id": "sentry:tvos-source",
+                "name": "iOS",
+                "type": "http",
+            },
+        ]
+        builtins.extend(json.loads(CUSTOM_SOURCE_CONFIG))
+        return builtins
+
+    @pytest.fixture
+    def reversed_alias_map(self):
+        return {"sentry:ios-source": "sentry:ios", "sentry:tvos-source": "sentry:ios"}
+
+    # Explicitly empty list of sources
+    @pytest.mark.django_db
```

Comment:
If you want to make your testcases faster, I'd remove this marker and use `sentry.testutils.helpers.override_options` ctx mgr since that appears to be the only source of DB access

Reply:
ooh, this is a great hint!